client/src/main/java/com/alibaba/nacos/client/config/filter/impl/ConfigRequest.java

```diff
@@ -22,11 +22,11 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import static com.alibaba.nacos.client.config.common.ConfigConstants.CONTENT;
-import static com.alibaba.nacos.client.config.common.ConfigConstants.DATA_ID;
-import static com.alibaba.nacos.client.config.common.ConfigConstants.GROUP;
-import static com.alibaba.nacos.client.config.common.ConfigConstants.TENANT;
-import static com.alibaba.nacos.client.config.common.ConfigConstants.TYPE;
+import static com.alibaba.nacos.client.config.filter.impl.ConfigMetadata.CONTENT;
```

Comment:
Why use new one? I think the old one has no problem.

Reply:
Because I don't want to  reference var of other module. It doesn't flow the p3c rule.

Reply:
The common module define these class and constants for all module used, which to reduce the duplicate codes and constants. 

These  constants both used in server and client. So I think you should use common ConfigConstants.

Reply:
Emmm, Yes. I think you are right. I will put all the common constants into the common module, but it's big project for me. So I want to do this one by one. I'm going to optimize all the modules then put extract common constants into the common module. It's my plan.

Reply:
Who can lead me that do this? I feel it too hard.😭